tests/prow_ci.sh

```diff
@@ -30,3 +30,163 @@ override_rosacli_build () {
   # go back to previous dir
   cd -
 }
+
+# configure_aws will setup aws credentials for the CI job run
+# It requires two parameters 
+# the first one for aws credential file path which is required
+# the second one for aws region which is required
+# the third one for second aws cred info for shared vpc scenario which is optional
+# usage: configure_aws <aws file path> <region> <shared vpc credential file>
+configure_aws () {
```

Comment:
maybe `configure_aws` - > `aws_config` could be better.

Reply:
`configure_aws` is an action about the function. I prefer to keep it.

Reply:
Based on your preferable.